Traducir.Api/Controllers/StringsController.cs

```diff
@@ -147,10 +147,10 @@ void ComposePredicate(Func<SOString, bool> newPredicate)
 
         [HttpGet]
         [Authorize]
-        [Route("app/api/suggestions-by-user/{userId:INT}")]
-        public async Task<IActionResult> GetSuggestionsByUser(int userId)
+        [Route("app/api/suggestions-by-user/{userId:INT}/{filterId:Int?}")]
+        public async Task<IActionResult> GetSuggestionsByUser(int userId, int? filterId)
```

Comment:
I wouldn't call this `filterId`... it's the suggestion state, so it should be `StringSuggestionState? suggestionState = null`